messages/context.json

```diff
@@ -0,0 +1,12 @@
+{
+  "shippingEstimate-bd": "Shipping estimates in business days",
```

Comment:
shouldn't the value be the key to the message id? e.g., in this case it should be
```
"shippingEstimate-bd": "shippingEstimate-bd"
```

Reply:
Wait, hahaha I don't think so. `context.json` should contain descriptions of what your message is for.

Reply:
ah, ok then :-)